### PR TITLE
[Please ignore] - Corrected the port number

### PR DESCRIPTION
**Pull request by shaikhwajeed**
This is to check MVP scenario end to end.

### CODE REVIEW

File: msteams-platform/tutorials/get-started-dotnet-app-studio.md [hunk 1]
```diff
@@ -67,10 +67,10 @@ After the repo is cloned, use Visual Studio to open the solution file **Microsof
 
 When the app starts, a browser window opens with the root of the app launched. You can go to the following URLs to verify that all the app URLs are loading:
 
-- `https://localhost:44327/`
-- `https://localhost:44327/hello`
-- `https://localhost:44327/first`
-- `https://localhost:44327/second`
+- `http://localhost:3978/`
```
Comment (Wajeed-msft) [inline msteams-platform/tutorials/get-started-dotnet-app-studio.md:70]: Could you please update this to https?

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

File: msteams-platform/tutorials/get-started-dotnet-app-studio.md (hunk 1 above)
Comment (shaikhwajeed) [reply msteams-platform/tutorials/get-started-dotnet-app-studio.md:70]: Completed.

**Comment by shaikhwajeed** [comment]
@Surbhigupta12 - could you please review this?

File: msteams-platform/tutorials/get-started-dotnet-app-studio.md (hunk 1 above)
Comment (v-rodriguezd) [reply msteams-platform/tutorials/get-started-dotnet-app-studio.md:70]: @shaikhwajeed, please update the comment given by @Wajeed-msft

**Comment by surbhigupta12** [comment]
This is a test PR. Hence, closing it.

**Pull request closed without merging**